src/Models/FileLink.php

```diff
@@ -30,7 +30,14 @@ public function getCMSFields(): FieldList
 
     public function getDescription(): string
     {
-        return $this->File()?->getFilename() ?? '';
+        $file = $this->File();
+        if (!$file?->exists()) {
+            return _t('LinkField.FILE_DOES_NOT_EXIST', 'File does not exist');
```

Comment:
```suggestion
            return _t(__CLASS__ . '.FILE_DOES_NOT_EXIST', 'File does not exist');
```
Same with any other PHP translation strings you're adding.
Some of the pre-existing ones are not set up like this, but they will be. See https://github.com/silverstripe/silverstripe-linkfield/issues/140

Reply:
Updated